clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -139,7 +139,7 @@ class DeleteExecutor[CI <: ClioIndex](deleteCommand: DeleteCommand[CI])(
 
     val paths = metadata.pathsToDelete.to[immutable.Iterable]
     Source[URI](paths)
-      .mapAsyncUnordered(paths.size) { path =>
```

Comment:
Why were we doing this in the first place?

Reply:
The paths don't need to be reassembled in-order after this stage of the stream for the mainline code to work (scalamock is really being too strict).

Reply:
supposedly scalamock as an implicit inAnyOrder but that seems to not be the case with AsyncMockFactory vs MockFactory